repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -406,4 +407,4 @@ def produce_messages(tasks):
                                            to_remove=to_remove_pkgs))
 
     if to_enable_repos:
-        api.produce(RepositoriesSetupTasks(to_enable=to_enable_repos))
+        api.produce(CustomTargetRepositories(repos=[CustomTargetRepository(repoid=repo) for repo in to_enable_repos]))
```

Comment:
that's the thing. You are presenting RH repos as custom which is wrong. You are changing here the meaning of CustomTargetRepositories modesl at all.